packages/node-opentelemetry/src/otel-logger/loglayer.ts

```diff
@@ -0,0 +1,100 @@
+import {
+  LoggerlessTransport,
+  type LogLayerTransportParams,
+  LogLevel,
+  type LogLevelType,
+} from '@loglayer/transport';
+import { Attributes, diag } from '@opentelemetry/api';
+
+import type { LoggerOptions } from './';
+import { Logger } from './';
+
+// Map LogLayer levels to their numeric values for comparison
+const LOG_LEVEL_PRIORITY = {
+  [LogLevel.trace]: 0,
+  [LogLevel.debug]: 1,
+  [LogLevel.info]: 2,
+  [LogLevel.warn]: 3,
+  [LogLevel.error]: 4,
+  [LogLevel.fatal]: 5,
+};
+
+export type HyperDXLogLayerOptions = LoggerOptions & {
+  apiKey?: string;
+  getCustomMeta?: () => Attributes;
+  maxLevel?: LogLevelType;
+};
+
+export default class HyperDXLogLayer
```

Comment:
For more info on how this transport is implemented:

https://loglayer.dev/transports/creating-transports.html